scripts/flask/web_interface.py

```
@@ -412,6 +528,7 @@ def run_script_with_live_output(script_name, arg_values=[]):
             bufsize=1,  # Line-buffered output
             env=os.environ.copy()
         )
+        logger.info("Running script: %s %s", script_path, arg_values)
```

Ruff (0.11.9)

Undefined name `logger`

(F821)